# Data Science Tools And Ecosystem

In This Notebook, Data science Tools and Ecosystem Are Summarize



## OBJECTIVES ##

- List of popular languages of Data Scientists.
- List the commonly used libraries by Data Scientists.
- List out some Data Sciences Tools.
- Do some simple arithmetics expression.


Some of the popular languages used by Data scientists are
1. Phython.
2. Java.
3. R and RStudio.
4. SQL.
5. C++.
6. Scala.
7. Julia. and many more languages

Some of the commonly used Libraries by Data Scientists includes:
1. Python. 
2. Pandas.
3. Numpy.
4. Matplotlib.
5. Seaborn. 

Data Sciense Tools:
1. Jupyter.
2. Jupyterlab.
3. Apache Zeppelin.
4. RStudio.
5. SPYDER.

### Below are a few example of evaluating arithmetic expressions in Python:


This is a simple arithmetic expression to multiply then add integers:

In [7]:
(3*4)+5

17

This will convert 200 minutes to hours by dividing by 60:

In [9]:
200/60 

3.3333333333333335

There are  3.3 hours in 200 minutes

## AUTHOR: 
HAPPINESS IJEOMA